lib/ansible/modules/system/group.py

```diff
@@ -98,7 +99,8 @@ class Group(object):
     GROUPFILE = '/etc/group'
 
     def __new__(cls, *args, **kwargs):
-        return load_platform_subclass(Group, args, kwargs)
+        new_cls = get_platform_subclass(Group)
+        return super(cls, new_cls).__new__(new_cls)
```

Comment:
I believe this should be
```suggestion
        return super(cls, new_cls).__new__(new_cls, *args, **kwargs)
```
per my above comment.